Modules/Blueprints/src/selxBlueprintImpl.cxx

```diff
@@ -397,4 +399,363 @@ ::Write( const std::string filename )
     boost::get( &ComponentPropertyType::parameterMap, this->m_Graph ) ),
     make_edge_label_writer( boost::get( &ConnectionPropertyType::parameterMap, this->m_Graph ) ) );
 }
+
+BlueprintImpl::ParameterValueType
+BlueprintImpl::VectorizeValues(ComponentOrConnectionTreeType & componentOrConnectionTree)
+{
+  std::string        propertySingleValue = componentOrConnectionTree.data();
+  ParameterValueType propertyMultiValue;
+  for (auto const & value : componentOrConnectionTree)
+  {
+    propertyMultiValue.push_back(value.second.data());
+  }
+  if (propertyMultiValue.size() > 0)
+  {
+    if (propertySingleValue != "")
+    {
+      throw std::invalid_argument("XML tree should have either 1 unnamed element or multiple named properties");
+    }
+  }
+  else
+  {
+    propertyMultiValue.push_back(propertySingleValue);
+  }
+  return propertyMultiValue;
+}
+
+/*
```

Comment:
I see why this function is not need anymore but do we have to save it here in a large comment? It will be in the history in any case.

Reply:
You are right it is a left-over. Will clean up.